pymc3/backends/tracetab.py

```diff
@@ -23,11 +23,11 @@ def trace_to_dataframe(trace, chains=None, flat_names=None):
     if flat_names is None:
         flat_names = {v: create_flat_names(v, shape)
                       for v, shape in var_shapes.items()
-                      if not v.endswith('_')}
+                      if not hide_transformed_vars or not v.endswith('_')}
```

Comment:
equivalent to the more readable: `if not (hide_transformed_vars and v.endswith('_'))`